webapi/src/main/scala/org/knora/webapi/routing/v2/ListsRouteV2.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright © 2015-2018 the contributors (see Contributors.md).
+ *
+ * This file is part of Knora.
+ *
+ * Knora is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Knora is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.routing.v2
+
+import akka.actor.ActorSystem
+import akka.event.LoggingAdapter
+import akka.http.scaladsl.server.Directives.{get, path, _}
+import akka.http.scaladsl.server.Route
+import akka.stream.ActorMaterializer
+import akka.util.Timeout
+import org.knora.webapi.messages.v2.responder.listsmessages.{ListsGetRequestV2, NodeGetRequestV2}
+import org.knora.webapi.routing.{Authenticator, RouteUtilV2}
+import org.knora.webapi.util.StringFormatter
+import org.knora.webapi.{ApiV2WithValueObjects, BadRequestException, IRI, SettingsImpl}
+
+import scala.concurrent.ExecutionContextExecutor
+
+/**
+  * Provides a function for API routes that deal with search.
+  */
+object ListsRouteV2 extends Authenticator {
+
+    def knoraApiPath(_system: ActorSystem, settings: SettingsImpl, log: LoggingAdapter): Route = {
+        implicit val system: ActorSystem = _system
+        implicit val executionContext: ExecutionContextExecutor = system.dispatcher
+        implicit val timeout: Timeout = settings.defaultTimeout
+        implicit val stringFormatter: StringFormatter = StringFormatter.getGeneralInstance
+        implicit val materializer = ActorMaterializer()
+        val responderManager = system.actorSelection("/user/responderManager")
+
+        path("v2" / "list" / Segment) { lIri: String =>
```

Comment:
`v2/list` should be `v2/lists` to be in line with the convention we use.

Reply:
done in https://github.com/dhlab-basel/Knora/pull/852/commits/0d527e5f659e9e867595b0d2ea313ee668cc8086